catalog-rest-service/src/main/java/org/openmetadata/catalog/util/ChangeEventParser.java

```diff
@@ -77,7 +78,7 @@ public static String getEntityUrl(ChangeEvent event) {
     if (Objects.nonNull(urlInstance)) {
       String scheme = urlInstance.getScheme();
       String host = urlInstance.getHost();
-      return String.format("<%s://%s/%s/%s|%s>", scheme, host, event.getEntityType(), fqn, fqn);
+      return String.format("<%s://%s/%s/%s>", scheme, host, event.getEntityType(), fqn);
```

Comment:
why are we changing this? was it wrong before?

Reply:
updated , it needed to be changed for teams